packages/terra-worklist-data-grid/src/orion-fusion-theme/ColumnHeaderCell.module.scss

```diff
@@ -8,12 +8,12 @@
     // Pinned Columns
     --terra-worklist-data-grid-pinned-columns-header-border: 1px solid #c8cacb;
     --terra-worklist-data-grid-pinned-columns-header-box-shadow: 1px 0 #c8cacb;
-    
+
     // Pinned Columns divider
     --terra-worklist-data-grid-pinned-columns-divider-background-color: #808080;
     --terra-worklist-data-grid-pinned-columns-divider-box-shadow: 0;
-    
+
     // Hover
-    --terra-worklist-data-grid-hover-background-color: #d5e7f2;
+    --terra-worklist-data-grid-header-hover-background-color: #d5e7f2;
```

Comment:
Is the header supposed to have a different hover color?

Reply:
The variable was renamed by a different story in terra-default-theme. This change ensures same variable is used in the other two themes.